src/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode, useEffect } from 'react'
+
+interface Props {
+  src: string
+  onLoad: () => void
+}
+
+export default function ScriptLoader({ src, onLoad }: Props): ReactNode {
+  useEffect(() => {
+    const scripts: HTMLScriptElement[] = Array.from(document.querySelectorAll('script'))
+    if (scripts.find((script) => script.src === src)) {
+      onLoad()
+      return
+    }
+    const script: HTMLScriptElement = document.createElement('script')
+    script.src = src
+    script.onload = onLoad
+    document.body.appendChild(script)
+  }, [src, onLoad])
```

Comment:
Remove the dependency on `src, onLoad`. You only want this to fire once, ever.

Even in the old code, `componentDidMount` was only triggered once

Reply:
Should I override the eslint rule for this? I did it based on this comment within this discussion:
https://github.com/facebook/react/issues/14920#issuecomment-470913287

Reply:
Yes you should override for this specific case